audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/model/AuditErrorMessage.java

```diff
@@ -55,11 +54,6 @@ private AuditErrorMessage(
         this.error = Optional.ofNullable(error).orElseThrow(() -> new IllegalArgumentException("Error cannot be null"));
     }
 
-    @Generated
-    public Throwable getError() throws JsonProcessingException {
-        return MAPPER.treeToValue(error, Throwable.class);
-    }
-
     @JsonIgnore
```

Comment:
I think we need a single standard for the message getters.  I would suggest we turn it back to what it was and on a separate ticket agree on a standard.

Reply:
The reason this was removed is because when parsing the json value for the error it may contain error types from other services that the audit service does not have access to.
Worth having a discussion though.

Reply:
There is a different solution in the PolicyRequest for the getters.  It throws an uncaught exception which is ok, but it is with an exception that belongs to Spring and that is a non-starter for the messages.  I dig into this as it relates to code that I am dealing with as well.  In the mean time this is not a major point and should not slow us down here.